dev/com.ibm.ws.openapi.ui/swagger-ui/webpack.config.js

```diff
@@ -21,6 +21,9 @@ module.exports = {
     }
   },
   devtool: "hidden-source-map",
+  optimization: {
+     minimize: true
+  },
```

Comment:
Why was this needed? This used to be the default when you build in `production` mode, has that changed?

Reply:
This is to make it easier for us in the future if we want to get the unminized version back for when we are checking whether a listed library (licensing or security question) is actually in the code we ship. to get the minimized version both devtool and this setting need to be changed to get the unminimized version